TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/samHWmarkers.java

```diff
@@ -0,0 +1,73 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Servo;
+
+import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
+import org.firstinspires.ftc.teamcode.trajectorysequence.TrajectorySequence;
+
+/*
+ * This is an example of a more complex path to really test the tuning.
+ */
+@Autonomous(group = "drive")
+public class samHWmarkers extends LinearOpMode {
+    Servo testServo;
+    @Override
```

Comment:
Looks good, good thinking for having override in the right spot.

Reply:
wow thanks